aws/data_source_aws_redshift_service_account_test.go

```diff
@@ -33,6 +33,6 @@ data "aws_redshift_service_account" "main" { }
 
 const testAccCheckAwsRedshiftServiceAccountExplicitRegionConfig = `
 data "aws_redshift_service_account" "regional" {
-	region = "eu-west-1"
+	region = "eu-west-2"
```

Comment:
Is there a reason to change this test?

Reply:
Possibly not - I'm new to updating the terraform source code

I thought it would help validate the new Redshift service account (eu-west-2) being added - remove if not required

Reply:
No worries, was just wondering :)